docs/document/content/reference/faq/_index.en.md

```diff
@@ -341,3 +341,47 @@ The followings are core codes from ProxoolDataSource getConnection method in `Pr
 ```
 
 For more alias usages, please refer to [Proxool](http://proxool.sourceforge.net/configure.html) official website.
+
+## 29. [Other] The property settings in the configuration file do not take effect when integrating ShardingSphere with Spring Boot 2.x ?
+
+Answer:
+
+Note that the property name in the Spring Boot 2.x environment is constrained to allow only lowercase letters, numbers and short transverse lines, ` [a-z] [0-9] ` and ` - `.
```

Comment:
@cheese8 `[a-z] [0-9]` and `-` may be better. No extra spaces are required within the symbol ``.